android-patches/patches/JniUtils/ReactCommon/react/renderer/components/switch/Android.mk

```diff
@@ -0,0 +1,11 @@
+--- a/ReactCommon/react/renderer/components/switch/Android.mk
```

Comment:
Curious to know which SO file do these fabric code end up it ?

Reply:
It goes into libfabricjni.so